package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "basics": "mocha */test",
-    "lint": "eslint */lib/*.jsx */test/*.spec.js",
+    "lint": "eslint \"**/*.js\"",
```

Comment:
If you don't want to accept this eslint config, I suggest at least keeping this eslint pattern fix. Even against master this will catch currently missed errors.